testutil/singlevm/setup.sh

```diff
@@ -305,6 +294,23 @@ echo "$identity" >> "$ciao_env"
 identity_url=$(echo $identity | sed 's/^.*=//')
 "$ciao_bin"/run_image.sh $identity_url &> /dev/null
 
+sleep 1
```

Comment:
No right now.  The is just one of the many race conditions in single vm that we'll need to address at some point.  The problem is that without the sleep the ciao-cli image commands that follow will fail as they will be executed before the ciao-image service has started.  We might be able to fix this if we use systemd to launch the ciao services but this will need to be done as part of a bigger change.